src/proj_internal.h

```diff
@@ -107,6 +107,13 @@ void pj_inherit_ellipsoid_def (const PJ *src, PJ *dst);
 void pj_erase_ellipsoid_def (PJ *P);
 int pj_calc_ellipsoid_params (PJ *P, double a, double es);
 
+
```

Comment:
Why 2 blank lines?

Reply:
Not for any good reason - will eat one of them